lips/lip-10.md

```diff
@@ -0,0 +1,231 @@
+---
+lip: 10
+title: Libra ID Spec
+authors: Andrey Chursin(@andll), Kevin Hurley (@kphfb)
+status: Draft
+type: Standard
+created: 11/03/2020
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Summary
+
+This LIP describes Libra ID - the human-readable identifier for user accounts.   
+
+In addition to describing the format of the Libra ID, this LIP also describes the off-chain protocol for the peer-2-peer payment with Libra ID.
+
+This document does not describe an off-chain protocol for using Libra ID for merchant use case - this protocol will be part of merchant flow LIP.  
+
+# Motivation
+Libra ID provides a convenient way to identify a user's account within a VASP. There are two main use cases for Libra ID:
+* For peer-2-peer payments, Libra ID allows a user to provide a human-readable ‘address’ to the sender. Basically, instead of sending a [lip-5 address](https://github.com/libra/lip/blob/master/lips/lip-5.md) (that needs to be refreshed for each payment, ideally), the receiver can just share their Libra ID once. In this case Libra ID plays the role of an email address for payments. Benefits for this use case:
+  * Libra ID is a user-friendly identifier, unlike lip-5 addresses
+  * The Libra ID resolution process can facilitate the exchange of rich information(e.g. nicknames, profile picture, etc.) about both the sender and receiver of the payment
+  * Libra ID provides better privacy, comparing to using sub-addresses, because Libra ID is not shared on the chain
+  * Libra ID is a persistent user identifier. By contrast, a sub-address is an ephemeral user identifier that should be recycled frequently to preserve privacy.
+* For the merchant use case, Libra ID allows users to self-identify to merchants. In this case, Libra ID plays the role of credit card number or also can be seen as a variant of OpenID-type protocol.
+
+# Libra ID format
+Libra ID is a string in format `<user-id>$<libra-id-domain>`.
+
+* **user-id**, identifies an account within a VASP:
+  * Case insensitive
+  * Regex: `^[a-zA-Z0-9][a-zA-Z0-9-_+.]?$`
```

Comment:
That regexp looks wrong. I think you want a "*" instead of a "?". Otherwise, the user-ids can only be 1 or 2 characters in length.

Reply:
Good call!